lacework/resource_lacework_integration_gar.go

```diff
@@ -192,11 +198,12 @@ func resourceLaceworkIntegrationGarCreate(d *schema.ResourceData, meta interface
 	data := api.NewContainerRegistry(d.Get("name").(string),
```

Comment:
I noticed we're doing a lot of type assertions without changing for an error. I think we should be checking for errors for all of these assertions.

Reply:
Update: Discussed this and understand that the schema validates the type so there is no need for checking errors on the type assertions.